index.html

```diff
@@ -239,6 +284,12 @@ <h1 class="foot-title">
                 required
               />
             </li>
+            <li class="input-full-name">
+                        <input type="text" name="first_name" id="first_name" maxlength="30" placeholder="First name"
+                            required>
+                        <input type="text" name="last_name" id="last_name" maxlength="30" placeholder="Last name"
+                            required>
+                    </li>
```

Comment:
- [ ] The contact section on the desktop lacks left and right padding as seen in the Figma design. Please fix this.

#### Your design
![image](https://user-images.githubusercontent.com/84156714/235982310-bb641995-463e-4104-9a74-605e0d86a9af.png)

#### Figma Design
![image](https://user-images.githubusercontent.com/84156714/235982664-c473aa05-e741-4c6c-a69d-070a30dec083.png)